### PR TITLE
Add error handling for bookmark delete function.

### PR DESCRIPTION
**Pull request by josephhhhhhhhh**

### CODE REVIEW

File: src/main/java/seedu/eduke8/ui/Ui.java [hunk 1]
```diff
@@ -393,7 +393,7 @@ public void deleteNoteInteractions(TopicList topicList) {
 
         printMessage(DELETE_NOTE_PROMPT_FOR_TOPIC);
         String topicName = SCANNER.nextLine();
```
Comment (anqi20) [inline src/main/java/seedu/eduke8/ui/Ui.java]: Is it good to put a .trim() here?

**Review by anqi20 — APPROVED** [review]

File: src/main/java/seedu/eduke8/ui/Ui.java (hunk 1 above)
Comment (josephhhhhhhhh) [reply src/main/java/seedu/eduke8/ui/Ui.java]: Done!

File: src/main/java/seedu/eduke8/command/BookmarkCommand.java [hunk 2]
```diff
@@ -13,43 +14,49 @@
...
+        if (bookmarkList.getCount() == 0) {
+            throw new Eduke8Exception("You currently do not have any bookmarks stored.");
+        }
     }
 
-    public BookmarkCommand(int deleteIndex, String typeOfBookmarkCommand, BookmarkList bookmarks) {
+    public BookmarkCommand(int deleteIndex, String typeOfBookmarkCommand, BookmarkList bookmarkList)
+            throws Eduke8Exception {
+        if (bookmarkList.getCount() == 0) {
+            throw new Eduke8Exception("You currently do not have any bookmarks stored.");
```
Comment (kstonekuan) [inline src/main/java/seedu/eduke8/command/BookmarkCommand.java:47]: Just need to abstract this magic string and should be good

Comment (kstonekuan) [reply src/main/java/seedu/eduke8/command/BookmarkCommand.java:47]: Remember to add the junits for this as well

Comment (josephhhhhhhhh) [reply src/main/java/seedu/eduke8/command/BookmarkCommand.java:47]: Good catch, will do so right away.

**Pull request merged**